src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -303,6 +304,15 @@ private void saveUser(User entity, Map<String, String> okapiHeaders, PostgresCli
                 "This barcode has already been taken"))));
           return;
         }
+        if (isUserTypeNotPopulatedError(reply)) {
+          logger.info("The user type was not populated for the user with id {}", entity.getId());
```

Comment:
We already have similar error log in method isUserTypePopulated()
So this info log can be deleted